Finalize v0.4.3

sirosen opened the pull request:
Includes the fix for #63, a version bump, and a rebuild of the docs.

There was more that I wanted to get into this version, but it also has been pushed back further than I anticipated.

Is there anything more that we demand for v0.4.3?

corpulentcoffee commented:
This PR isn't off the very current tip of `master`, so it might be missing the doc change in #76...?

sirosen commented:
@corpulentcoffee I've rebased to capture those changes, and also added a Makefile which does many useful things. Importantly, there's `make help`, which I stole from Sphinx as an idea, and very much like.
